pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -208,6 +211,50 @@ func (buildSpec *BuildSpec) StrategyName() string {
 	return buildSpec.Strategy.Name
 }
 
+// VulnerabilityIgnoreOptions refers to ignore options for vulnerability scan
+type VulnerabilityIgnoreOptions struct {
+
+	// Issues references the security issues to be ignored in vulnerability scan
+	Issues []string `json:"issues,omitempty"`
+
+	// Severity indicates the severities of security issues to be ignored (comma separated)
+	Severity string `json:"severity,omitempty"`
+
+	// IgnoreUnfixed indicates flag to display only fixed vulnerabilities
+	Unfixed bool `json:"unfixed,omitempty"`
+}
+
+// VulnerabilityScanOptions references the options for vulnerability scanning
+type VulnerabilityScanOptions struct {
+
+	// Enabled indicates whether to run vulnerability scan for image
+	Enabled bool `json:"enabled,omitempty"`
+
+	// FailPush indicates whether to push the image if the vulnerability scan fails
+	FailPush bool `json:"fail,omitempty"`
```

Comment:
Nit: On second thought, I am a bit unhappy with the name. It is not wrong or anything, but I am somewhat feel it should be maybe called more like `FailOnFinding` or something.

Also, the JSON field name needs to be revisited based on https://github.com/shipwright-io/community/issues/186#issuecomment-1927111254.

Reply:
Changed it to `FailOnFinding `